renderer/components/Autopay/Autopay.js

```diff
@@ -13,8 +13,10 @@ import messages from './messages'
 
 const customiseTheme = theme => {
   return createThemeVariant('autopilot', {
```

Comment:
Not related to this PR obviously, but i wonder why do we call theme `autopilot`. Should be `autopay`

Reply:
Good catch. This isn't the first time this typo has slipped its way in somewhere. Fixed.